pkg/controller/controller.go

```diff
@@ -16,3 +32,31 @@ func AddToManager(m manager.Manager) error {
 	}
 	return nil
 }
+
+// EventListner ...
+func EventListner() {
+	kubeconfig := os.Getenv("KUBECONFIG")
```

Comment:
These logic need not be handled by us as operator-sdk lib does these low level stuff.
Once this code is inside the controller file, we can start to interact with informers directly.

Reply:
okay!!